db/migrate/20150728194147_create_event_blacklists.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEventBlacklists < ActiveRecord::Migration
+  def change
+    create_table :event_blacklists do |t|
+      t.string  :event_name
+      t.string  :provider_model
+      t.bigint  :ems_id
+      t.boolean :enabled
```

Comment:
The filters will be exposed in the UI and we want to allow users to easily enable/disable filters without having to delete the item.  The event monitor is only getting the enabled filters.